web/wp-content/themes/bumblebee/template-parts/content-search.php

```diff
@@ -10,26 +10,26 @@
 ?>
 
 <article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
-	<header class="entry-header">
-	<?php the_title( sprintf( '<h2 class="entry-title"><a href="%s" rel="bookmark">', esc_url( get_permalink() ) ), '</a></h2>' ); ?>
+	<div class="content-image">
+		<?php bumblebee_post_thumbnail( 'large' ); ?>
+	</div>
+	<div class="content-container">
+		<header class="entry-header">
+		<?php
+		if ( 'post' === get_post_type() ) {
+			$categories_list = get_the_category_list( esc_html__( ', ', 'bumblebee' ) );
+			if ( $categories_list ) {
+				echo '<span class="cat-links">';
+				/* translators: 1: list of categories. */
+				printf( esc_html__( '%1$s', 'bumblebee' ), $categories_list ); // phpcs:ignore WordPress.Security.EscapeOutput
```

Comment:
Hey @fariasf - this is the stupid function I CANNOT squeeze through the validator, could you take a look please?